ui/src/components/common/Breadcrumbs/index.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { Breadcrumbs } from "./index";
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useLocation: () => ({
+    pathname: "/",
+  }),
+}));
+
+describe("Breadcrumbs", () => {
+  it("loads pipeline screen", () => {
+    render(
+      <BrowserRouter>
+        <Breadcrumbs />
+      </BrowserRouter>
+    );
+    expect(screen.getByTestId("namespace-breadcrumb")).toBeInTheDocument();
+    expect(screen.getByTestId("mui-breadcrumbs")).toBeInTheDocument();
+  });
+});
+
+// jest.mock("react-router-dom", () => ({
```

Comment:
What is the reason to keep these commented lines?

Reply:
have to fix an issue in the test cases(commented code), Shakira requested to create a branch for simultaneous testing.